### PR TITLE
Feature/requester secret

### PR DESCRIPTION
**Pull request by PierreJeanjacquot**
add `--resquesterSecret <index=value..>` option for `test` and `run` commands

- for `test`: injects the secrets in the simulated worker environment
- for `run`: provisions the secrets in the SMS with unique uuid and map them in requestorder `params.iexec_secrets`

Example:

```sh
iapp test --resquesterSecret 1=foo 42=bar
# injects envs IEXEC_REQUESTER_SECRET_1=foo and IEXEC_REQUESTER_SECRET_42=bar
```

### CODE REVIEW

File: cli/README.md [hunk 1]
```diff
@@ -65,6 +65,8 @@ Options:
   (use quotes to provide multiple args).
 - use `--inputFile <url...>` to provide one or more input files to your iApp
   during testing.
+- use `--requesterSecret <key=value...>` to provide one or more requester
+  secrets to your iApp during testing.
```
Comment (cedric25) [inline cli/README.md:69]: What do you think of having here the same description as the one we have in the CLI itself? (the one added in `src/index.js`)

The two first options `--args` and `--inputFile` are kind of ok to understand, but `--requesterSecret` is more cryptic for a non-iExec developer. I wouldn't assume anyone getting to this README understand what it is.
 - Briefly explain it here?
 - Include link to https://protocol.docs.iex.ec/for-developers/technical-references/application-io#requester-secrets ?
 - Give a meaningful example: "For an iApp like web3mail, requester secrets include "emailSubject" or "senderName".

**Review by cedric25 — COMMENTED** [review]
All good for me.
Haven't really tested it, waiting for necessary code to add in the iApp template.

**Comment by PierreJeanjacquot** [comment]
> All good for me. Haven't really tested it, waiting for necessary code to add in the iApp template.

Possible to test with https://github.com/iExecBlockchainComputing/iapp/pull/34

File: cli/README.md (hunk 1 above)
Comment (PierreJeanjacquot) [reply cli/README.md]: added links to doc ddc80a06ac0ac30b6e46814259c81676bbca47fb

**Pull request merged**